README.md

```diff
@@ -100,13 +108,55 @@ adding an amplifier. *Only go in this direction after familiarizing yourself
 with allowances of radio transmissions in your area on your frequency of
 interest.*
 
-### Transmit!
+#### Nvidia Jetson Series (experimental)
+(*Please read the external hardware for Raspberry Pi above first.)   
+On Jetson, the external hardware setup is slightly different from Raspberry Pi. 
+
+We use hardware PWM pin as a frequency output pin. Unlike the Raspberry Pi, we need only one output pin 
+because we modulate the signal by switching (on-off keying) instead of attenuating.
+And we use only one register (4.7kΩ) for the Jetson.
+It's because when we tested it on the Jetson Nano model, the voltage of the output pin was ~1.5V and the signal was too weak if I use 2x4.7kΩ registers like the Raspberry Pi.
+
+The output pin varies by the Jetson model. Check the following table.
+|Devices|Output pin (Board numbering)|
+|-------|----------------------------|
+|Jetson Xavier, Clara AGX Xavier|18|
+|Other devices|33|
 
+Here's the full schematic of the external hardware for Jetson Series:
+Schematic                      | Real world
+-------------------------------|------------------------------
+![](img/schematic-jetson.jpg)   |![](img/jetson-nano.jpg)
```

Comment:
Not a biggie, but the 'real world picture' would be even better if the viewer could better see where the pins are connected on the Pin header. So more taking a picture from the right side so that you can see onto the full pin-header and see which pins are connected while still having the watch in the foreground.